data/json/snippets/effect_on_conditions.json

```diff
@@ -222,7 +244,15 @@
       "a confident voice saying \"That which consumes your world poisons you against us.  Ignore its lies, and abandon your half digested reality.\"",
       "a confident voice saying \"Have you never dreamed of being one with everything?  Out here is the end of limitations like separation or alone.\"",
       "a confident voice saying \"The breakdown of order has danger, but also opportunity.  There is still time to influence the end result, come outside and add your will to the mix.\"",
-      "a confident voice saying \"This world is half eaten, but there are others yet untouched in the infinte.  It is unlikely you will find the way through the chaos out here to them, but much more likely than you surviving in this dead world.\""
+      "a confident voice saying \"This world is half eaten, but there are others yet untouched in the infinite.  It is unlikely you will find the way through the chaos out here to them, but much more likely than you surviving in this dead world.\"",
+      "the membrane of reality trembling as if someone was rhythmically beating a drum, and the beats are arranged in a looping message evidently broadcasted in Morse code: \"-.. . .- - …. / ..- .--. --- -. / -.-- --- ..-\"",
```

Comment:
Why do the eldritch horrors know morse code?

Reply:
Ramza designed the voices to originate from the Netherum and according to the faction design document nether realm is imprinted by anything that crossed it. Thus those are not voices from other dimensions but of the unknown entities of the nether. In my belief it is why one can understand them at all, as human crossing has left enough imprint to allow this one sided communication. This would include languages, concepts, ideas, emotions, parts of culture etc. that are now used in reverse to relay a message from beyond. This is why such thing as Morse code could be used as a way to communicate. Of course alien concepts are imprinted too thus the cosmic horror, and cryptic and not always coherent context of the message.

Unless I missed it, Ramza didn't explain why the entities want you outside exposed to the portal storm, but my guess is that the imprinting property means knowledge, shape and opportunity. Shadow needs the light to exist and an object that would cast it.